cassovary-core/src/main/scala/com/twitter/cassovary/graph/TestGraph.scala

```diff
@@ -76,14 +76,22 @@ object TestGraphs {
     NodeIdEdgesMaxId(14, Array())
     ).iterator, StoredGraphDir.BothInOut)
 
-  val nodeSeqIterator = () => Seq(
-      NodeIdEdgesMaxId(10, Array(11, 12, 13)),
-      NodeIdEdgesMaxId(11, Array(12, 14)),
-      NodeIdEdgesMaxId(12, Array(14)),
-      NodeIdEdgesMaxId(13, Array(12, 14)),
-      NodeIdEdgesMaxId(14, Array(15)),
-      NodeIdEdgesMaxId(15, Array(10, 11))
-      ).iterator
+  val nodeSeq =  Seq(
+    NodeIdEdgesMaxId(10, Array(11, 12, 13)),
+    NodeIdEdgesMaxId(11, Array(12, 14)),
+    NodeIdEdgesMaxId(12, Array(14)),
+    NodeIdEdgesMaxId(13, Array(12, 14)),
+    NodeIdEdgesMaxId(14, Array(15)),
+    NodeIdEdgesMaxId(15, Array(10, 11)))
+
+  val nodeSeqWithEmpty = Seq(
+    NodeIdEdgesMaxId(0, Array.empty[Int]),
+    NodeIdEdgesMaxId(1, Array.empty[Int])
+  ) ++ nodeSeq
+
+
+  val nodeSeqIterator = () => nodeSeq.iterator
```

Comment:
bear in mind that this will slightly be changed once https://github.com/twitter/cassovary/pull/137 is merged. pls merge master in before your next PR.